bedrock/mozorg/templates/mozorg/home/home-en.html

```diff
@@ -38,15 +38,17 @@
 {% block page_css %}
   {{ css_bundle('home-en') }}
 
-  {% if switch('firefox-daylight-promo-banner') and not switch('fundraising-banner-eoy2020') %}
+  {% if switch('firefox-daylight-promo-banner') %}
     {{ css_bundle('daylight-promo-banner') }}
   {% elif switch('fundraising-banner-eoy2020') %}
     {{ css_bundle('fundraising-banner') }}
   {% endif %}
 {% endblock %}
 
 {% block page_banner %}
-  {% if switch('firefox-daylight-promo-banner') and not switch('fundraising-banner-eoy2020') %}
+  {% if switch('vpn-promo-banner', ['de', 'fr']) %}
```

Comment:
I don't think this should be in the EN template.

Reply:
Ok, I pushed a commit with some changes. It's a little less DRY, but easier to follow I think?

Reply:
Oh you're right, I forgot the others extend EN as the base... so I guess either approach works but less repetition is probably better after all. The locale part of condition means it can never load for EN anyway so no risk of it appearing in English.